packages/components/src/internal/components/base/Grid.tsx

```diff
@@ -276,7 +268,7 @@ class GridBody extends PureComponent<GridBodyProps> {
         );
     }
 
-    renderRow(row: any, r: number, highlight?: boolean): any {
+    renderRow = (row: any, r: number, highlight?: boolean): ReactNode => {
```

Comment:
Minor: `renderRow` is now the only fn that uses '=' constructor, maybe align them for consistency